scripts/integration-tests/e2e-prettier.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+#==============================================================================#
+#                                  SETUP                                       #
+#==============================================================================#
+
+# Start in scripts/integration-tests/ even if run from root directory
+cd "$(dirname "$0")" || exit
+root="$PWD"
+
+source utils/local-registry.sh
+source utils/cleanup.sh
+
+# Echo every command being executed
+set -x
+
+# Clone prettier
+git clone --depth=1 https://github.com/prettier/prettier /tmp/prettier
+cd /tmp/prettier || exit
```

Comment:
```suggestion
git clone --depth=1 https://github.com/prettier/prettier tmp/prettier
cd tmp/prettier || exit
```

Just to match the other e2e tests (it's a .gitignored folder)